src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/DefaultProjectSnapshotManagerTest.cs

```diff
@@ -659,10 +662,186 @@ public void SolutionClosing_ProjectChangedEventsCorrect()
         textLoader.Verify(d => d.LoadTextAndVersionAsync(It.IsAny<LoadTextOptions>(), It.IsAny<CancellationToken>()), Times.Never());
     }
 
+    [UIFact]
```

Comment:
I hope we can remove `[UIFact]` from all of these tests once the dispatcher thread is truly gone. That'd be a good test that the new approach is working.